README.md

```diff
@@ -298,6 +299,107 @@ expect(queryByText('Hello World')).toHaveStyle([{ color: 'black' }, { fontWeight
 expect(queryByText('Hello World')).not.toHaveStyle({ color: 'white' });
 ```
 
+### `toBeVisible`
+
+```typescript
+toBeVisible();
+```
+
+Check that the given element is visible.
+
+An element is visible if **all** the following conditions are met:
```

Comment:
It seems that we have a related task in RNTL for adding option to ignore hidden elements in queries. It seems that these two PRs should share the same logic for visibility check.
Pls check the other PR: https://github.com/callstack/react-native-testing-library/pull/1064/files#diff-bc7c1ed436746d189e6b39ab5fdd4e2092589eb23da9a67e68e067a884231d73

Reply:
This is no longer relevant, as I have learned implementing the [`isInaccessible`](https://github.com/callstack/react-native-testing-library/blob/cc52c610fc9b9f8686f8e6204454b3f6a13888b3/src/helpers/accessiblity.ts#L5) function for RNTL, that these two are similar but not the same, as e.g. `opacity: 0` does not exclude from accessibility tree, but would make the element invisible.